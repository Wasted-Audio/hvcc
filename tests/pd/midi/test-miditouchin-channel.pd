#N canvas 1013 321 503 302 12;
#X obj 97 100 print;
#X obj 97 61 touchin 2;
#X connect 1 0 0 0;
