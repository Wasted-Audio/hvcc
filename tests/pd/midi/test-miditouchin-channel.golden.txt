[@ 0.000] print: 0
[@ 0.000] print: 1
[@ 0.000] print: 2
[@ 0.000] print: 3
[@ 0.000] print: 4
[@ 0.000] print: 3
[@ 0.000] print: 2
[@ 0.000] print: 1